src/config/database.js

```diff
@@ -8,12 +8,16 @@ const database = async () => {
             : process.env.DATABASE;
 
     const client = new MongoClient(DATABASE, { useNewUrlParser: true, useUnifiedTopology: true });
+    const myDb = "blogAppDB";
+    const collectionOne = "users";
+    const collectionTwo = "blogs";
 
     try {
         await client.connect();
         logger.info('Connected to the database.');
-        const userCollection = client.db("blogAppDB").collection("users")
-        module.exports = userCollection;
+        const userCollection = client.db(myDb).collection(collectionOne);
+        const blogCollection = client.db(myDb).collection(collectionTwo);
```

Comment:
instead of naming collectionOne and collectionTwo , can you maintain a json where userCollection : users...